node/src/util.rs

```diff
@@ -83,3 +84,16 @@ pub fn get_triple_timeout() -> Duration {
         .unwrap_or_default()
         .unwrap_or(crate::types::PROTOCOL_TRIPLE_TIMEOUT)
 }
+
+pub fn is_elapsed_longer_than_timeout(timestamp_sec: u64, timeout: Duration) -> bool {
```

Comment:
that elapsed works with Instant, which records the relative timestamp in the system, not the absolute timestamp. Since we are sending the timestamp as a field in the messages, it needs to be comprehensible in all nodes' system, thus using u64 timestamp to represent the time.